addons/hr_evaluation/hr_evaluation.py

```diff
@@ -1,374 +1,456 @@
+        res['domain'] = str([('id', 'in', [meeting.id for meeting in meeting_ids])])
+        return res
+
+    @api.one
+    def log_meeting(self, meeting_subject, meeting_date):
+        message = _("Subject: %s <br> Meeting scheduled at '%s'<br>") % (meeting_subject, meeting_date.split(' ')[0])
+        return self.message_post(body=message)
+
+
+class hr_employee(models.Model):
+    _inherit = "hr.employee"
+
+    evaluation_date = fields.Date('Next Appraisal Date', help="The date of the next appraisal is computed by the appraisal plan's dates (first appraisal + periodicity).")
+    appraisal_manager = fields.Boolean('Manager')
+    apprasial_manager_ids = fields.Many2many('hr.employee', 'apprasial_manager_rel', 'hr_evaluation_evaluation_id')
+    apprasial_manager_survey_id = fields.Many2one('survey.survey', 'Manager Appraisal')
+    appraisal_colleagues = fields.Boolean('Colleagues')
+    appraisal_colleagues_ids = fields.Many2many('hr.employee', 'appraisal_colleagues_rel', 'hr_evaluation_evaluation_id')
+    appraisal_colleagues_survey_id = fields.Many2one('survey.survey', "Employee's Appraisal")
+    appraisal_self = fields.Boolean('Employee')
+    apprasial_employee = fields.Char('Employee Name')
+    appraisal_self_survey_id = fields.Many2one('survey.survey', 'Self Appraisal')
+    appraisal_subordinates = fields.Boolean('Collaborator')
+    appraisal_subordinates_ids = fields.Many2many('hr.employee', 'appraisal_subordinates_rel', 'hr_evaluation_evaluation_id')
+    appraisal_subordinates_survey_id = fields.Many2one('survey.survey', "collaborate's Appraisal")
+    appraisal_repeat = fields.Boolean('Periodic Appraisal', default=False)
+    appraisal_repeat_number = fields.Integer('Repeat Every', default=1)
+    appraisal_repeat_delay = fields.Selection([('year', 'Year'), ('month', 'Month')], 'Repeat Every', copy=False, default='year')
+    appraisal_count = fields.Integer(compute='_appraisal_count', string='Appraisal Interviews')
+
+    @api.one
+    def _appraisal_count(self):
+        Evaluation = self.env['hr_evaluation.evaluation']
+        self.appraisal_count = Evaluation.search_count([('employee_id', '=', self.id)])
+
+    @api.onchange('appraisal_manager')
+    def onchange_manager_appraisal(self):
+        self.apprasial_manager_ids = [self.parent_id.id]
+
+    @api.onchange('appraisal_self')
+    def onchange_self_employee(self):
+        self.apprasial_employee = self.name
+
+    @api.onchange('appraisal_colleagues')
+    def onchange_colleagues(self):
+        if self.department_id.id:
+            self.appraisal_colleagues_ids = self.search([('department_id', '=', self.department_id.id), ('id', '!=', self.parent_id.id)])
+
+    @api.onchange('appraisal_subordinates')
+    def onchange_subordinates(self):
+        manager = set()
+        for emp in self.search([]):
+            if emp.parent_id:
+                manager.add(emp.parent_id.id)
+        self.appraisal_subordinates_ids = list(manager)
```

Comment:
collaborator = poeple the employee is manager
So domain should be [('parent_id', '=', self.id)]